assets/samples/cloudNativeFullStack.yaml

```diff
@@ -99,32 +105,46 @@ spec:
       #     cpu: 300m
       #     memory: 1.5Gi
 
-      # Optional: disables automatic restarts of OneAgent pods in case a new version is available. True
-      # by default.
+      # Optional: Adds arguments to add to the OneAgent installer
+      # For a list of available options, see https://www.dynatrace.com/support/help/shortlink/linux-custom-installation
+      # For a list of the limitations for OneAgents in Docker, see https://www.dynatrace.com/support/help/shortlink/oneagent-docker#limitations
+      #
+      # args: []
+
+      # Optional: Adds additional environment variables to OneAgent pods
+      #
+      # env: []
+
+      # Optional: Disables automatic restarts of OneAgent pods in case a new version is available
+      # Defaults to "true"
       #
       # autoUpdate: true
 
-      # Optional: DNS Policy for OneAgent pods. Defaults to ClusterFirstWithHostNet.
-      # See more: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+      # Optional: Sets the DNS Policy for OneAgent pods
+      # Defaults to "ClusterFirstWithHostNet"
+      # For more information on DNS policies, see https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
       #
       # dnsPolicy: "ClusterFirstWithHostNet"
 
-      # Optional: labels are customer defined labels for OneAgent Pods to structure workloads as desired.
+      # Optional: Defines labels added to OneAgent pods
+      # Can be used to structure workloads
       #
       # labels:
       #   custom: label
 
-      # Optional: additional environment variables to add to the OneAgent Pods.
+      # Optional: Sets the URI for the image containing the OneAgent installer used by the DaemonSet
+      # Defaults to the latest OneAgent image on the tenant's registry
       #
-      # env: []
+      # image:
 
-      # Optional: arguments to add to the OneAgent installer.
-      # Available options: https://www.dynatrace.com/support/help/shortlink/linux-custom-installation
-      # Limitations: https://www.dynatrace.com/support/help/shortlink/oneagent-docker#limitations
+      # Optional: If specified, indicates the OneAgent version to use
+      # Defaults to "latest"
+      # This property expects the version to be in the semver format
```

Comment:
Nitpick: can a property expect something?

"The version has to be provided in semver format."